.gitignore

```diff
@@ -22,6 +22,9 @@ docs/source/api/utils
 docs/source/api/plugins
 docs/source/api/optional-plugins
 docs/source/guides/writer/libs/data/vmimage
+.venv
```

Comment:
Please remove those changes from your commit, this is not related to avocado-framework and it should be part of your [local .gitignore](https://medium.com/@peter_graham/how-to-create-a-local-gitignore-1b19f083492b)

Reply:
Oh I see, fixed